File: send_email.php
<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $name = $_POST['name'];
    $email = $_POST['email'];
    $phone = $_POST['phone'];
    $message = $_POST['message'];

    $to = "sales@coomiexpress.com";

    $subject = "New Contact Form Submission";

    $email_content = "Name: $name\n";
    $email_content .= "Email: $email\n";
    $email_content .= "Phone Number: $phone\n";
    $email_content .= "Message:\n$message\n";

    if (isset($_FILES['resume'])) {
        $file_name = $_FILES['resume']['name'];
        $file_tmp = $_FILES['resume']['tmp_name'];

        // Add the uploaded file as an attachment
        $attachment = chunk_split(base64_encode(file_get_contents($file_tmp)));
        $email_content .= "--mixedBoundary\n";
        $email_content .= "Content-Type: application/octet-stream; name=\"$file_name\"\n";
        $email_content .= "Content-Disposition: attachment\n";
        $email_content .= "Content-Transfer-Encoding: base64\n\n";
        $email_content .= "$attachment\n";
    }

    $headers = 'From: ' . $email;

    // Send the email
    mail($to, $subject, $email_content, $headers);

    header("Location: contact.html?success=1");
} else {
    header("Location: contact.html?success=0");
}
?>

File: send_email.py
from flask import Flask, request, render_template
import smtplib
from email.mime.text import MIMEText
from email.mime.multipart import MIMEMultipart
import os

app = Flask(__name__)

# Load SMTP email server details from environment variables
SMTP_SERVER = os.environ.get('SMTP_SERVER', 'smtp.gmail.com')
SMTP_PORT = os.environ.get('SMTP_PORT', 587)
SMTP_USERNAME = os.environ.get('SMTP_USERNAME', 'hannahsooahlee2004@gmail.com')
SMTP_PASSWORD = os.environ.get('SMTP_PASSWORD', 'PASSWORD')
SENDER_EMAIL = os.environ.get('SENDER_EMAIL', 'hannahsooahlee2004@gmail.com')
RECEIVER_EMAIL = os.environ.get('RECEIVER_EMAIL', 'sales@coomiexpress.com')


@app.route('/')
def index():
    return render_template('index.html')


@app.route('/submit', methods=['POST'])
def submit_form():
    if request.method == 'POST':
        name = request.form['name']
        email = request.form['email']
        phone = request.form['phone']
        message = request.form['message']

        # Create a message
        subject = 'New Contact Form Submission'
        msg = MIMEMultipart()
        msg['From'] = SENDER_EMAIL
        msg['To'] = RECEIVER_EMAIL
        msg['Subject'] = subject
        msg.attach(MIMEText(
            f'Name: {name}\nEmail: {email}\nPhone: {phone}\n\nMessage:\n{message}', 'plain'))

        try:
            # Connect to the SMTP server and send the email
            server = smtplib.SMTP(SMTP_SERVER, SMTP_PORT)
            server.starttls()
            server.login(SMTP_USERNAME, SMTP_PASSWORD)
            server.sendmail(SENDER_EMAIL, RECEIVER_EMAIL, msg.as_string())
            server.quit()
            return 'Message sent successfully!'
        except Exception as e:
            return f'An error occurred: {str(e)}'


if __name__ == '__main__':
    app.run(debug=True)
